reactive-governance/terraform/modules/cas-reactive/main.tf

```diff
@@ -0,0 +1,305 @@
+/*
+Copyright 2024 Google LLC
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+
+provider "google" {
+  project = var.project_id
+  credentials = file("prj-22-sa.json")
+}
+
+# Enable Cloud Resource Manager API
+module "project-service-cloudresourcemanager" {
+  source  = "terraform-google-modules/project-factory/google//modules/project_services"
+  version = "4.0.0"
+
+  project_id = var.project_id
+
+  activate_apis = [
+    "cloudresourcemanager.googleapis.com"
+  ]
+}
+
+# Enable APIs
+module "project-services" {
+  source  = "terraform-google-modules/project-factory/google//modules/project_services"
+  version = "4.0.0"
+
+  project_id = var.project_id
+
+  activate_apis = [
+    "compute.googleapis.com",
+    "bigquery.googleapis.com",
+    "pubsub.googleapis.com",
+    "cloudscheduler.googleapis.com",
+    "cloudfunctions.googleapis.com",
+    "cloudasset.googleapis.com",
+    "cloudbuild.googleapis.com",
+    "run.googleapis.com",
+    "eventarc.googleapis.com"
+  ]
+  depends_on = [module.project-service-cloudresourcemanager]
+}
+
+## RESOURCES TO CREATE MISSING LABELS DASHBOARD ##
+# Create Pub/Sub topic to list projects in the parent node
+resource "google_pubsub_topic" "cas_topic" {
+  name       = var.cas_topic
+  depends_on = [module.project-services]
+}
+
+resource "google_cloud_scheduler_job" "cas_job" {
+  name        = var.scheduler_cas_job_name
+  description = var.scheduler_cas_job_description
+  schedule    = var.scheduler_cas_job_frequency
+  region      = var.region
+
+  pubsub_target {
+    topic_name = google_pubsub_topic.cas_topic.id
+    data       = base64encode("hello")
```

Comment:
Can this be removed, the `data` field is optional.

Reply:
Removed